lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
+    "packages/**"
+  ],
+  "version": "0.0.41"
```

Comment:
Does this update automatically with lerna?

Reply:
yup!